src/modules/shared/components/SideBarMenu/SideBarMenu.tsx

```diff
@@ -0,0 +1,147 @@
+import { useEffect, useState } from "react";
+import { Sidebar, Menu, MenuItem } from "react-pro-sidebar";
+import { NavLink } from "react-router-dom";
+import styles from "./SideBarMenu.module.css";
+
+const saveSideBarState = (state: { collapsed: boolean }) => {
+  localStorage.setItem("sideBarState", JSON.stringify(state));
+};
+const getSideBarState = () => {
+  const state = localStorage.getItem("sideBarState");
+  return state ? JSON.parse(state) : { collapsed: true };
+};
+
+const SideBarMenu = () => {
+  // const { loginData, removeLoginData } = useContext(AuthContext);
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(
+    getSideBarState().collapsed
+  );
+
+  const toggleCollapse = () => {
+    setIsCollapsed(!isCollapsed);
+  };
+
+  useEffect(() => {
+    saveSideBarState({ collapsed: isCollapsed });
+  }, [isCollapsed]);
+
+  return (
+    <div
+      className={`sidebar-container position-fixed d-flex ${
+        isCollapsed ? "collapsed" : ""
+      }`}
+    >
+      <Sidebar collapsed={isCollapsed}>
+        <Menu>
+          <MenuItem
+            onClick={toggleCollapse}
+            icon={
+              <div className={styles["arrow-menu-item-wrapper"]}>
+                <div className={styles["arrow-menu-item"]}>
+                  <svg
```

Comment:
The same goes here, we can take all svg icon in one file in the assets. ex: SideBarIcons.tsx, and in it include all component versions of the SVG.